code/modules/mob/living/carbon/carbon.dm

```diff
@@ -647,16 +647,26 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 	if(I)
 		SEND_SIGNAL(I, COMSIG_CARBON_TOGGLE_THROW, in_throw_mode)
 
+#define THROW_MODE_ICON 'icons/effects/cult_target.dmi'
+
 /mob/living/carbon/proc/throw_mode_off()
 	in_throw_mode = FALSE
 	if(throw_icon) //in case we don't have the HUD and we use the hotkey
 		throw_icon.icon_state = "act_throw_off"
+	if(!client)
+		return
+	if(client?.mouse_pointer_icon == THROW_MODE_ICON)
+		client.mouse_pointer_icon = initial(client.mouse_pointer_icon)
```

Comment:
you aready null check client here so you don't need to `?` sanity check it when accessing its vars. Alternatively you can just nuke the if(!client) check above

Reply:
fixed